tools/pyi/gen_pyi.py

```diff
@@ -1377,6 +1377,7 @@ def replace_special_case(hint: str) -> str:
             "float8_e4m3fnuz",
             "float8_e5m2",
             "float8_e5m2fnuz",
+            "float8_e8m0fnu",
```

Comment:
We should do the todo above... :D

Reply:
tracking in https://github.com/pytorch/pytorch/issues/146647